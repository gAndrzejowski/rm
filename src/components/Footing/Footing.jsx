import React from 'react';
import PageTitle from '../PageTitle/PageTitle';
import styles from './Footing.scss';

export default function Footing() {
    return (
        <footer>
            <PageTitle />
        </footer>
    );
}

