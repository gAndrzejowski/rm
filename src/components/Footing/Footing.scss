.appFooting {
  bottom:0;
  height:10vh;
  padding-left:5vw;
  display:flex;
  justify-content: flex-start;
  align-items:center;
}