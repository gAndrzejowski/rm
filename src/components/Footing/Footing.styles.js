const styles = {
  appFooting: {
    bottom: 0,
    height: '10vh',
    paddingLeft: '5vw',
    display: 'flex',
    justifyContent: 'flex-start',
    alignItems: 'center',
  },
};
export default styles;
