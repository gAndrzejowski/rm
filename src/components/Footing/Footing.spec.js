import React from 'react';
import renderer from 'react-test-renderer';
import Footing from './Footing';
import {shallow} from 'enzyme'

describe('Footing component', () => {
   it('renders correctly', () => {
       const tree = renderer.create(<Footing />).toJSON();
       expect(tree).toMatchSnapshot();
   })
});