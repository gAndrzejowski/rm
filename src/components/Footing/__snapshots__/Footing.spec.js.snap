// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footing component renders correctly 1`] = `
<footer>
  <p
    className={undefined}
  >
    netflixroulette
  </p>
</footer>
`;
